extension/src/popup/App.tsx

```diff
@@ -22,11 +18,6 @@ import { Router } from "./Router";
 
 import "./styles/global.scss";
 
-// .isBigNumber() not catching correctly, so checking .isBigNumber
-// property as well
-const isSerializable = (value: any) =>
```

Comment:
for some reason jest didn't like importing this from this file. So moved the func to a helper file

Reply:
Oh weird - I wonder if it had something to do with the other code in this file 🤔 Prob best to have it in a helper anyway